tests/contrib/django/asgi.py

```diff
@@ -0,0 +1,6 @@
+from django.core.asgi import get_asgi_application
+
+from ddtrace.contrib.asgi import TraceMiddleware
+
+
+application = TraceMiddleware(get_asgi_application())
```

Comment:
is this what we are recommending for customers to do now?

Reply:
A bit of a long story.... because of #2559 there's no way to be able to identify traces as coming from a particular route (no route tagging, no http propagation, no query params, etc) in order to filter traces that we don't care about (polling, startup, teardown, etc).

I added in the ASGI middleware to mitigate this for now so that we don't get a bunch of flaky `time.sleep()`s everywhere.

I think it's ok because the ASGI spans just wrap the Django ones. With #2656 we'll remove this explicit usage.